recipes/default.rb

```diff
@@ -41,14 +41,14 @@
   chef_gem 'slackr' do
     compile_time false if respond_to?(:compile_time)
   end
-  handler_file = "#{node['chef_handler']['handler_path']}/slack_handler.rb"
```

Comment:
If we're dropping chef_handler, do we also need to remove it from the dependencies in metadata.rb and stop including it in recipes?

Reply:
chef_handler is now a built-in resource in chef 14+ so there is no attribute for handler_path anymore as the chef_handler cookbook is deprecated and no longer recommended for use. That's also why I put in the major version bump in metadata. https://supermarket.chef.io/cookbooks/chef_handler

Reply:
so to answer your question yes.

Reply:
I'd recommend doing a version 0.9.2 verision bump to master with the changes you just merged in, and then we can remove them from this commit and we'll do a major 1.0.0 version bump. Users on older chef clients can then pin to ~>0.9 and users on chef 14+ can use latest 1.0.0.